iroha/src/bridge.rs

```diff
@@ -95,6 +96,19 @@ impl Identifiable for Bridge {
     type Id = BridgeId;
 }
 
+/// An entity used for storing data of any third-party transaction.
+///
+/// # Note
+///
```

Comment:
Did you missed a note?

Reply:
@humb1t oops... No, just forgot to remove